lib/cmr/cmr_client.rb

```diff
@@ -173,6 +173,25 @@ def delete_collection(provider_id, native_id, token)
       delete(url, {}, nil, headers.merge(token_header(token)))
     end
 
+    def ingest_variable(metadata, provider_id, native_id, token, content_type = nil)
+      # if native_id is not url friendly or encoded, it will throw an error so we check and prevent that
+      if Rails.env.development? || Rails.env.test?
+        url = "http://localhost:3002/providers/#{provider_id}/variables/#{encode_if_needed(native_id)}"
+      else
+        url = "/ingest/providers/#{provider_id}/variables/#{encode_if_needed(native_id)}"
+      end
+
+      headers = {
+        'Accept' => 'application/json',
+        'Content-Type' => "application/#{Rails.configuration.umm_version}; charset=utf-8"
```

Comment:
Do we want the umm_version here? Based on my conversations with CMR we are likely to have a new version and we'll start splitting these out into `umm_c_version` and `umm_v_version` -- either way, var isn't versioned yet.